src/components/network-chart/index.js

```diff
@@ -80,7 +82,16 @@ class NetworkChartImpl extends React.PureComponent<Props> {
   componentDidMount() {
     this.focus();
   }
-
+  componentDidUpdate(prevProps) {
+    if (
+      this.props.scrollToSelectionGeneration >
+      prevProps.scrollToSelectionGeneration
+    ) {
+      if (this._virtualListRef.current) {
```

Comment:
nit: you don't need this check, because it's done in `scrollSelectionIntoView` already